install_config/aggregate_logging.adoc

```diff
@@ -1518,3 +1518,627 @@ $ oc label node --all logging-infra-fluentd=true <1>
+
+* `exported-fields-Default`
+* `exported-fields-rsyslog`
+* `exported-fields-systemd`
+* `exported-fields-kubernetes`
+* `exported-fields-docker`
+* `exported-fields-pipeline_metadata`
+* `exported-fields-ovirt`
+* `exported-fields-aushape`
+* `exported-fields-tlog`
+
+[discrete]
+[[exported-fields-Default]]
+=== Top Level Fields
+
+The top level fields are common to every application, and may be present in every record.
+For the Elasticsearch template, this is what populates the actual mappings of `default` in
+the template's mapping section.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `@timestamp`
+| The UTC value marking when the log payload was created, or when the log payload
+was first collected if the creation time is not known. This is the log
+processing pipeline’s best effort determination of when the log payload was
+generated. The `@` prefix convention to note a field as being reserved for a
+particular use. With Elasticsearch, most tools look for `@timestamp` by default.
+For example, the format would be 2015-01-24 14:06:05.071000.
+
+| `geoip`
+|This is geo-ip of the machine.
+
+| `hostname`
+|The FQDN of the entity generating the original payload, and this field is an attempt to derive this context. Sometimes the entity generating it knows the context. While other times that entity has a restricted namespace itself, which is known by the collector or normalizer.
+
+| `ipaddr4`
+|The IP address V4 of the source server, which can be an array.
+
+| `ipaddr6`
+|The IP address V6 of the source server, if available.
+
+| `level`
+|The logging level as provided by `rsyslog` (severitytext property), python's
+logging module. Possible values are as listed at
+link:http://sourceware.org/git/?p=glibc.git;a=blob;f=misc/sys/syslog.h;h=ee01478c4b19a954426a96448577c5a76e6647c0;hb=HEAD#l74[`misc/sys/syslog.h`]
+plus `trace` and `unknown`. For example, "alert crit debug emerg err info notice
+trace unknown warning". Note that `trace` is not in the `syslog.h` list but many
+applications use it.
+
+You should only use `unknown` when the logging system gets a value it does not
+understand, and note that it is the highest level. Consider `trace` as higher,
+or more verbose, than `debug`. Convert `error` to `err`. Convert `panic` to
+`emerg`. And convert `warn` to `warning`.
+
+Numeric values from `syslog/journal PRIORITY` can usually be mapped using the
+priority values as listed at
+link:http://sourceware.org/git/?p=glibc.git;a=blob;f=misc/sys/syslog.h;h=ee01478c4b19a954426a96448577c5a76e6647c0;hb=HEAD#l51[misc/sys/syslog.h].
+
+Log levels and priorities from other logging systems should be mapped to the
+nearest match. See
+link:https://docs.python.org/2.7/library/logging.html#logging-levels[python
+logging] for an example.
+
+| `message`
+|A typical log entry message, or payload. It can be stripped of metadata pulled
+out of it by collector/normalizer, that is UTF-8 encoded.
```

Comment:
should this be "collector or normalizer" as above (no slash). And probably needs an article.